### PR TITLE
feat: translated mobile banking app to hindi

### LINKED ISSUE #407: Translate Mobile Banking App into HIndi
**Summary:**

Issue #403  is parent issue for translation. it is a sub issue that will track about the Hindi translation.

### PR DESCRIPTION
**Pull request by amanjeetsingh150**
Fixes #407

Please Add Screenshots If any UI changes.

 <br> 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by amanjeetsingh150** [comment]
@dilpreet96 @therajanmaurya please review.

**Comment by therajanmaurya** [comment]
@amanjeetsingh150 Before the review, Please revert Gradle version and Gradle tools version change. This issue does not contain this type of change. After changing let me know.

**Comment by amanjeetsingh150** [comment]
@therajanmaurya on it will submit the updated PR soon 👍

**Comment by amanjeetsingh150** [comment]
@therajanmaurya @dilpreet96 reverted the gradle tool and gradle versions.
Please review!

File: app/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -1,254 +1,375 @@
...
-    <string name="questionnaire">Questionnaire</string>
-    <string name="about_us">About Us</string>
-    <string name="saving_account_details">Saving Account Details</string>
-    <string name="savings_account">Savings Account</string>
-    <string name="error_loan_account_details_loading">Error loading in loan accounts detail</string>
-    <string name="nominal_interest_rate">Nominal Interest Rate</string>
-    <string name="account_number">Account Number </string>
-    <string name="account_balance">Account Balance</string>
-    <string name="total_deposits">Total Deposits</string>
+    <string name="app_name">सेल्फ सर्विस ऐप</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values-hi/strings.xml]: @amanjeetsingh150 Please don't translate app name

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by therajanmaurya** [comment]
@amanjeetsingh150 Please make the above change first,  I want this PR merge first.

**Comment by amanjeetsingh150** [comment]
@therajanmaurya changed the app name back to English. Please review.

**Review by therajanmaurya — APPROVED** [review]

**Pull request merged**